.gitignore

```diff
@@ -46,3 +46,5 @@ uboot-commands.ubt
 
 NOTES
 /server.PID
+
+!/.idea/
```

Comment:
Why do we not want to ignore this? There also is a conflicting entry already.

Reply:
we have some run configurations we want to share (like the one which allows you to set breakpoints in clion and use a proper debugger) but you're right, it should only include that and not everything in the .idea dir

Reply:
(this is what clion automatically does when you add a run config to git)

Reply:
It should be partially or completely ignored, not unignored indeed.